### PR TITLE
Some minor fixes

### PR DESCRIPTION
**Pull request by carolinepacheco**
Some minor fixes.

### CODE REVIEW

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 1]
```diff
@@ -4346,15 +4346,18 @@ image::Fetch_Satellite_Images.png[align=center]
 |===
 | *Variable name* | *Description* | *Type*
 | `HOST_SITES`
-| Defines an utility to search and download satellite images.
+| Defines a host site to search and download satellite images.
```
Comment (mklkun) [inline src/docs/MLOS/MLOSUserGuide.adoc]: site or website? Check please.

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 2]
```diff
@@ -4346,15 +4346,18 @@ image::Fetch_Satellite_Images.png[align=center]
...
 | List [Peps, Copernicus, All] (default=Peps)
 | `OUTPUT_PATH`
 | Specifies the path where the data should be downloaded.
 | String
 |===
 
 
-NOTE: Please access link:https://peps.cnes.fr/rocket/#/register[Peps] and link:https://scihub.copernicus.eu/dhus/#/self-registration[Copernicus] website to create a new user account.
+
+NOTE: Please click https://peps.cnes.fr/rocket/#/register[here^] to create a new user account to acesss Peps webisite.
```
Comment (mklkun) [inline src/docs/MLOS/MLOSUserGuide.adoc]: webisite => website

Comment (mklkun) [inline src/docs/MLOS/MLOSUserGuide.adoc]: to create a new user account to => to create a new user account and ...

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 3]
```diff
@@ -4346,15 +4346,18 @@ image::Fetch_Satellite_Images.png[align=center]
...
+
+NOTE: Please click https://scihub.copernicus.eu/dhus/#/self-registration[here^] to create a new user account to acesss Copernicus webisite.
```
Comment (mklkun) [inline src/docs/MLOS/MLOSUserGuide.adoc]: Same:
account to => account and
webisite => website

File: src/docs/MLOS/MLOSUserGuide.adoc (hunk 1 above)
Comment (mboussaa) [reply src/docs/MLOS/MLOSUserGuide.adoc]: host domain ?

**Pull request merged**